### PR TITLE
Awsome books with ES6 PR

### PR DESCRIPTION
**Pull request by rachaouldbabaali**
in this PR, I have : 

- [x] Added book.js as a module.
- [x] Added datetime using luxon.
- [x] Fixed linters errors

### CODE REVIEW

File: modules/book.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  // getbook  method
+  static getBooks() {
+    let books;
+    if (localStorage.getItem('books') === null) {
+      books = [];
+    } else {
+      books = JSON.parse(localStorage.getItem('books'));
+    }
+    // console.log(books);
```
Comment (pinkmoon25) [inline modules/book.js]: Good job using ES6 syntax. However, please remove unused commented code from the codebase it is not the best practice to leave commented code.

File: modules/book.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    return books;
+  }
+
+  // addbook  method
+  static addBook(book) {
+    const books = Book.getBooks();
+    books.push(book);
+    localStorage.setItem('books', JSON.stringify(books));
+    // console.log(books);
```
Comment (pinkmoon25) [inline modules/book.js]: Please remove unused commented code from the codebase it is not the best practice to leave commented code

File: index.js [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+});
+
+document.querySelector('#bookOfList').addEventListener('click', (e) => {
+  Book.deleteBook(e.target);
+  Book.removeBook(e.target.previousElementSibling.textContent);
+});
+
+// Get current date and display it
+const date = DateTime.now().toLocaleString(DateTime.DATE_FULL);
+document.querySelector('#date').textContent = date;
```
Comment (pinkmoon25) [inline index.js]: [OPTIONAL] I think if we are using the luxon library it would be better if we also show the time along with the date for the user

**Review by pinkmoon25 — CHANGES_REQUESTED** [review]
Hi @rachaouldbabaali , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
Linters :heavy_check_mark: 
GitHub flow :heavy_check_mark: 
Professional Readme :heavy_check_mark:
ES6 syntax used :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by TadesseDev — APPROVED** [review]
### Hi @rachaouldbabaali 

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewer. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**